ui/app/src/components/PreviewDropTargetsPanel/PreviewDropTargetsPanel.tsx

```diff
@@ -127,67 +156,117 @@ export function PreviewDropTargetsPanel() {
     errorSelector: (source) => null
   });
 
+  return awaitingGuestCheckIn ? (
+    <Alert severity="info" variant="outlined" icon={<HourglassEmptyRounded />} sx={{ border: 0 }}>
+      <FormattedMessage defaultMessage="Waiting for the preview application to load." />
+    </Alert>
+  ) : allowedContentTypes.length ? (
+    listMode ? (
+      <>
+        <ListSubheader>
+          <FormattedMessage defaultMessage="Compatible types" />
+        </ListSubheader>
+        {allowedContentTypes?.map((contentType: ContentType, i: number) => {
+          return (
```

Comment:
Use simplified syntax
```suggestion
        {allowedContentTypes?.map((contentType: ContentType, i: number) => (
```